spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -72,4 +72,135 @@
       expect(page).to have_current_path(idv_hybrid_handoff_url)
     end
   end
+
+  describe 'navigating to How To Verify from Agreement page in 50/50 state' do
+    context 'opt in false at start but true during navigation' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
```

Comment:
Since `in_person_proofing_enabled` is always set to true, it could move up to a before block at the top of the describe and only happen once to reduce noise.